app/helpers/cookies_policy_helper.rb

```diff
@@ -7,7 +7,7 @@ def render_cookie_entry(cookie_name, cookie_desc, cookie_domain = nil)
   end
 
   def matomo_iframe_src
-    Spree::Config.matomo_url\
+    "#{Spree::Config.matomo_url}"\
```

Comment:
Interesting. I tested that in the console, because I wasn't familiar with this syntax and it worked for me. :man_shrugging:

Reply:
you can see this in the console if you try with a variable. \ and + don't have the same behaviour:

2.1.5 :036 > x = "x"
 => "x"
2.1.5 :037 > "111"\
2.1.5 :038 >   "222"
 => "111222"
2.1.5 :039 > x +
2.1.5 :040 >   "222"
 => "x222"
2.1.5 :041 > x.to_s +
2.1.5 :042 >   "222"
 => "x222"
2.1.5 :043 > x \
2.1.5 :044 >   "222"
NoMethodError: undefined method `x' for main:Object
	from (irb):43
	from /Users/luisramos/.rvm/rubies/ruby-2.1.5/bin/irb:11:in `<main>'
2.1.5 :045 > x.to_s \
2.1.5 :046 >   "222"
ArgumentError: wrong number of arguments (1 for 0)
	from (irb):45:in `to_s'
	from (irb):45
	from /Users/luisramos/.rvm/rubies/ruby-2.1.5/bin/irb:11:in `<main>'
2.1.5 :047 > "#{x}" \
2.1.5 :048 >   "222"
 => "x222"